### DISCUSSION TITLE
Does the Linear Elliptical Slice Sampler support batch sampling?

### DISCUSSION THREAD

**Discussion by mozjay0619** [meta-pytorch/botorch#2158] [Q&A]
The [description](https://botorch.org/api/_modules/botorch/utils/probability/lin_ess.html#LinearEllipticalSliceSampler) of the inputs does not have batch_size. Does the elliptical slice sampler not support batch sampling yet?

**Comment by saitcakmak** [comment]
Hi @mozjay0619. That's correct, it does not appear to support batch sampling yet. It is listed [among the ideas for future improvements](https://github.com/pytorch/botorch/blob/main/botorch/utils/probability/lin_ess.py?fbclid=IwAR2MXH3uOzGfuTzOUy5RTk32myU9nunAJJ3xdsvxl3jw3EEMwT1V9EpiTVI#L48-L49), though I don't know if anyone is planning to work on it. cc @SebastianAment
